apps/ledger-live-mobile/src/families/tron/accountActions.tsx

```diff
@@ -1,23 +1,18 @@
 import React from "react";
 import { Trans } from "react-i18next";
-import type { Account } from "@ledgerhq/types-live";
 import type { TronAccount } from "@ledgerhq/live-common/families/tron/types";
 import { NavigatorName, ScreenName } from "~/const";
 import { ActionButtonEvent, NavigationParamsType } from "~/components/FabActions";
 import { isAccountEmpty } from "@ledgerhq/live-common/account/index";
-import invariant from "invariant";
 import { TRX } from "../../../../../libs/ui/packages/crypto-icons/native";
 
 const getMainActions = ({
   account,
   parentAccount,
 }: {
   account: TronAccount;
-  parentAccount: Account;
+  parentAccount: TronAccount;
 }): ActionButtonEvent[] => {
-  const { tronResources } = account;
-  invariant(tronResources, "tron resources not parsed");
-
```

Comment:
This test should pass, that means we have an issue with our data model, a `TronAccount` should always have `tronResources` we even have a test for that that I wrote 3 years ago as my first topic at Ledger https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/families/tron/synchronisation.integration.test.ts#L53